File: Python/linearsearch.py
# Linear Search Implementation in Python

def linear_search(arr, x):
    for i in range(len(arr)):
        if arr[i] == x:
            return i
    return -1


arr = [2, 3, 4, 10, 40]
x = 10
result = linear_search(arr, x)
if result == -1:
    print("Element is not present in array")
else:
    print("Element is present at index", result)

File: Java/LinearSearch.java
// Linear Search in Java

public class LinearSearch {
    public static void main(String[] args) {
        // Array of integers
        int[] arr = { 2, 3, 4, 10, 40 };
        // Element to be searched
        int x = 10;
        // Linear Search
        System.out.println(linearSearch(arr, x));

    }

    static int linearSearch(int[] arr, int x) {
        // Linear Search
        int n = arr.length;
        for (int i = 0; i < n; i++) {
            if (arr[i] == x) {
                return i;
            }

        }
        return -1;

    }
}

// Output:
// 3